catroid/src/main/java/org/catrobat/catroid/pocketmusic/note/NoteName.java

```diff
@@ -87,6 +99,16 @@ public static int calculateDistanceToMiddleLineCountingSignedNotesOnly(MusicalKe
 		return calculateDistanceCountingNoneSignedNotesOnly(key.getNoteNameOnMiddleLine(), noteName);
 	}
 
+	public static List<NoteName> getAllPossibleOctaveStarts() {
+		List<NoteName> octaveStaringNotes = new ArrayList<>();
+		for (NoteName note : NoteName.values()) {
+			if (note.octaveStart) {
+				octaveStaringNotes.add(note);
+			}
+		}
+		return octaveStaringNotes;
+	}
```

Comment:
Since we only have 8 octaves, just put this in a static array.
Even if you want to keep the flexibility to want to add another octave or two later, octaves are 12 notes long.
So a for loop over multiples of 12 starting at 0 gives you all your C.

Reply:
I kept it dynamic so to make it easy to change the number of octaves.
Just as a not, there are currently 7 full octaves + 3 notes in the front and another C note at the back.